.gitignore

```diff
@@ -104,4 +104,5 @@ ENV/
 .DS_Store
 .vscode/settings.json
 custom_assets/*
-training/training_output/*
\ No newline at end of file
+training/training_output/*
+*-model-building-code.zip
```

Comment:
Should this also be added to `.Dockerignore`?

Reply:
Not need since `training/*` is already in Dockerignore. This zip file is inside that folder (if at all).